federation-js/src/service/printCoreSchema.ts

```diff
@@ -228,7 +245,11 @@ function printFederationTypeDirectives(type: GraphQLObjectType): string {
 }
 
 function printInterface(type: GraphQLInterfaceType, options?: Options): string {
-  // Federation change: print `extend` keyword on type extensions.
+  // TODO: I can't figure out why this is here. I wrote this 8 months ago. Nothing
+  // in the history explains the reason this is needed. When do we have an `extend type`
+  // in the final composed schema?
```

Comment:
I can't think of a reason either...

Reply:
Figured it out. This is a copy/pasta from `printFederatedSchema` which I must've leaned on for some inspiration while putting together the csdl printer.
https://github.com/apollographql/federation/blob/main/federation-js/src/service/printFederatedSchema.ts#L201-L210

So this makes sense where it was originally written, but I'll remove it from both composed and core printers.

Reply:
eadfe94